Mathlib/Algebra/Homology/Opposite.lean

```diff
@@ -221,6 +222,82 @@ instance (K : HomologicalComplex Vᵒᵖ c) (i : ι) [K.HasHomology i] :
 
 variable {V c}
 
+lemma quasiIsoAt_opFunctor_map_iff
```

Comment:
I think these could also be good `@[simp]` lemmas. But I'll leave that up to you.
```suggestion
@[simp]
lemma quasiIsoAt_opFunctor_map_iff
```